scRNA-seq-advanced/04-gene_set_enrichment_analysis.Rmd

```diff
@@ -223,19 +223,21 @@ gsea_results <- GSEA(geneList = lfc_vector,  # ordered ranked gene list
 Let's take a look at the GSEA results.
 
 ```{r view_gsea, live = TRUE, eval = FALSE}
-View(gsea_results@result |>
-       dplyr::arrange(dplyr::desc(NES))
+View(
```

Comment:
do we really benefit from `View()` here vs just printing it if it's just a table?